clients/gateway/gateway.go

```diff
@@ -47,7 +64,10 @@ func newTestServer() *httptest.Server {
 
 		// empty request: "{}"
 		emptyReqLen := 4
-		if len(b) <= emptyReqLen {
+		if string(b) == "null" {
```

Comment:
Is there a test that pass null or you added it just in case?

Reply:
This one does https://github.com/NethermindEth/juno/pull/956/files#diff-e6160c31d5a80924a41c26c6844e59e31250f3ddd0257d370cf6fdfb20a5019fR43-R46